### PR TITLE
Send XTZ signatures as buffer

### PR DESCRIPTION
**Pull request by argjv**
Signature verification was failing in the platform because the signature was not a buffer.

TICKET: BG-19801

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -120,11 +120,6 @@
     protobufjs "^6.8.9"
     tronweb "^2.7.2"
 
-"@bitgo/statics@^3.4.2":
-  version "3.4.4"
-  resolved "https://registry.yarnpkg.com/@bitgo/statics/-/statics-3.4.4.tgz#acc581980d30fa93e38460e2f9ea72beb60cfa91"
-  integrity sha512-rEXKec+K6ZuRCVQju/fH6/+7SOhvVtehyJZavEeytB0y5Ah9urL1lVXaXZGuKwPQaGqo4KGOjmOjGfxwkLRAMw==
-
```
Comment (CapnMigraine) [inline yarn.lock:127]: is this an intended change? it doesn't seem to be related

Comment (argjv) [reply yarn.lock:127]: Good catch. It is probably from the `yarn install` I run. I'll leave it to the BRE team to update it.
